configure.ac

```diff
@@ -147,6 +147,12 @@ AC_ARG_WITH([swift-toolchain],
    case $target_os in
       linux*)
 	    os_string="linux"
+	    case $target_cpu in
```

Comment:
Because it gets it wrong.  Much earlier in the evolution of swift on arm, we decided to canonicalize the representation of armv7 to "armv7" rather than the alternate variant "armv7l"  You'll find similar conversions other places.  This simply brings libdispatch inline with other components.